File: test/resources/hello_data/python/hello.py
'''
Created on 17/07/2013

@author: drodri
'''
import sys, os


def hello():
    #bii://data.bmp
    #bii://data.xml
    file_path = os.path.join(os.path.dirname(__file__), "data/data.bmp")
    with open(file_path, 'r') as fin:
        content = fin.read()
        sys.stdout.write(content.rstrip())
    file_path = os.path.join(os.path.dirname(__file__), "data/data.xml")
    with open(file_path, 'r') as fin:
        content = fin.read()
        sys.stdout.write(content.rstrip())

File: test/resources/hello_data/node/hello.js

/**
 * New node file
 */
var fs = require("fs");
var path = require("path");
var hello = function hello() {
	//bii://data.bmp
	//bii://data.xml
    process.stdout.write(fs.readFileSync(path.resolve(__dirname, "data/data.bmp")).toString()); 
    process.stdout.write(" ");
    process.stdout.write(fs.readFileSync(path.resolve(__dirname, "data/data.xml")).toString()); 
};
exports.hello = hello;
